matter/wifi/wf200/host_if.cpp

```diff
@@ -135,7 +135,7 @@ static void wfx_events_task_start()
   /* create an event group to track Wi-Fi events */
   sl_wfx_event_group = xEventGroupCreate();
 
-  if (xTaskCreate(wfx_events_task, "wfx_events", 1024, NULL, 1, &wfx_events_task_handle) != pdPASS) {
+  if (xTaskCreate(wfx_events_task, "wfx_events", STACK_SIZE_1K, NULL, TASK_PRIORITY_1, &wfx_events_task_handle) != pdPASS) {
```

Comment:
`STACK_SIZE_1K` I would give have the name of the task as part of the define name

`TASK_PRIORITY_1` see earlier comment

Reply:
done as per recommendation